kits/arm/ex_mobile_device_control.py

```diff
@@ -98,8 +93,18 @@ def get_ik(xyz_target, rot_target, ik_seed):
   return kin.solve_inverse_kinematics(ik_seed_pos, endeffector_position_objective(xyz_target), endeffector_so3_objective(rot_target))
 
 
+
+
+counter = 0
 # Startup
 while not abort_flag:
+  # Update mobile io state
+  state = m.getState()
+  print("counter: " + str(counter))
+  print("state: " + str(state))
+  counter += 1
+  if counter % 100 == 0:
+      print("counter: " + str(counter))
```

Comment:
@iamtesch , should we leave all/some of these print statements in?

Reply:
I don't think so -- I think the `counter` was part of a debugging process? We should clean up the temporary/debug code where possible

Reply:
I don't think so...